com.w1hkj.fldigi.yml

```diff
@@ -44,13 +43,19 @@ modules:
   - name: metainfo
     buildsystem: simple
     build-commands:
-        - install -D com.w1hkj.fldigi.metainfo.xml  /app/share/metainfo/com.w1hkj.fldigi.metainfo.xml
+        - install -D com.w1hkj.fldigi.metainfo.xml /app/share/metainfo/com.w1hkj.fldigi.metainfo.xml
         - install -D com.w1hkj.fldigi.png /app/share/icons/hicolor/128x128/apps/com.w1hkj.fldigi.png
+        - install -D com.w1hkj.flarq.png  /app/share/icons/hicolor/128x128/apps/com.w1hkj.flarq.png
         - install -D com.w1hkj.fldigi.desktop /app/share/applications/com.w1hkj.fldigi.desktop
+        - install -D com.w1hkj.flarq.desktop  /app/share/applications/com.w1hkj.flarq.desktop
```

Comment:
This is not a correct name. The correct name should be `com.w1hkj.fldigi.flarq.desktop` (and same thing, just with the .png suffix for the icon name), as I mentioned before.

Reply:
ok